ext_mongodb.php

```diff
@@ -715,17 +720,43 @@ function __debugInfo() : array;
 {
 	use DenySerialization;
 
-	public function __construct(private string $code, private ?mixed $scope = NULL)
+	private $code;
+
+	public function __construct(string $code, private ?mixed $scope = NULL)
 	{
+		if ( strstr( $code, "\0" ) !== false )
+		{
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "Code cannot contain null bytes" );
+		}
+		$this->code = $code;
```

Comment:
It is, because of the visibility modifier in the constructor. If you add "public", "protected" or "private", it adds an automatic variable declaration, and does the assignment for you.

So the code is correct.